api/lib/domain/models/AreaForRelease.js

```diff
@@ -19,5 +21,19 @@ module.exports = class AreaForRelease {
     this.competenceAirtableIds = competenceAirtableIds;
     this.color = color;
     this.frameworkId = frameworkId;
+
+    this.title_i18n = _computeTitleForI18N({ titleFrFr, titleEnUs });
   }
 };
+
+function _computeTitleForI18N({ titleFrFr, titleEnUs }) {
+  const title_i18n = {};
+  for (const locale of Object.values(LOCALE)) {
```

Comment:
J'avoue ne pas comprendre pour on itére la dessus, on a toute l'information dans le nom des paramètres. On pourrait simplifier en ayant simplement un {
fr: titleFrFr,
en: titleEnUs
}

Reply:
ok je voulais faire semblant d'un truc déjà un peu industrialisé, mais OK